docker-compose.yml

```diff
@@ -1,9 +1,9 @@
-version: "3.1"
+version: "4.4.3"
```

Comment:
This shouldn't change. It's the compose version.

Reply:
lol true sorry got ahead of myself didnt think